toffy/fov_watcher_test.py

```diff
@@ -14,6 +14,8 @@
 from toffy.test_utils import WatcherCases, RunStructureTestContext, RunStructureCases
 from toffy.fov_watcher import start_watcher
 from toffy.watcher_callbacks import build_callbacks
+from toffy.json_utils import read_json_file
```

Comment:
There's a couple files where you import both but don't use both. Only need to import the functions that are actually used